File: dataset_reduzido/codenet/Python/Code/s557080041.py
N = input()
if N == 0:
  print('Yes')
else:
  N = str(N)
  L = list(N)
  L = [int(s) for s in L]
  X = sum(L)
  if X % 9 == 0:
    print('Yes')
  else:
    print('No')

File: output/StarCoder/codenet/Python/Java/s557080041.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        String input = scanner.nextLine();
        
        int N;
        try {
            N = Integer.parseInt(input);
        } catch (NumberFormatException e) {
            System.out.println("No");
            return;
        }

        if (N == 0) {
            System.out.println("Yes");
        } else {
            int X = 0;
            while (N > 0) {
                X += N % 10;
                N /= 10;
            }
            if (X % 9 == 0) {
                System.out.println("Yes");
            } else {
                System.out.println("No");
            }
        }
    }
}